lib/components/menu/menu-sticker.tsx

```diff
@@ -41,6 +41,13 @@ const MenuSticker = () => {
         <div className="sticker">
           <div className="inner">
             <Tabs height="46px" value={tabValue} onChange={val => setTabValue(val)}>
+              <Tabs.Item
+                height="46px"
+                font="14px"
+                py={0}
+                label={isChinese ? '主页' : 'Home'}
+                value=""
+              />
```

Comment:
We can hide the Tabs on this page?

Reply:
I don't think there is a reason, it is much simpler to navigate like this.

Reply:
If we don't hide the Tabs, the Home page is a bit redundant.
`Home` is not a placeholder or a skeleton, I don't need to stay on the home page, because there is no point.
Make `Home` a brand home page and ask users to interact with it, which will enhance the value of the page and reduce the distraction of Tabs from the brand page presentation.

A simple comparison:
- Every time I open the home page I need to stay on the Home page, but there is nothing on this page and I still click on the tabs to go to the page I need, which only annoys me.
- The home page is a clean display of the brand without the distraction of Tabs or Footer, I can choose a category to browse from the brand page, which makes me feel that home is meaningful, it's like the cover of a book rather than a title page that forces people to browse.

Reply:
The home screen is indeed a brand page, which encourages you to use Geist. The primary button on the page is the ``Documentation`` button. 
While the cards were designed to show the Geist's features, I can see how those can be clickable. But in terms of UX, I don't think it's worth hiding the menu, users know it and it appears on all other pages.
I suggest using both, what do you think? @unix